playbooks/adhoc/s3_registry/s3_registry.yml

```diff
@@ -13,6 +13,8 @@
   vars:
     aws_access_key: "{{ lookup('env', 'S3_ACCESS_KEY_ID') }}"
     aws_secret_key: "{{ lookup('env', 'S3_SECRET_ACCESS_KEY') }}"
+    aws_bucket_name: "{{ aws_bucket | default(clusterid + '-docker') }}"
```

Comment:
http://jinja.pocoo.org/docs/dev/templates/ recommends using '~' instead of '+' for string concatenation, so:
```
aws_bucket_name: "{{ aws_bucket | default(clusterid + '-docker') }}"
```
should be:
```
aws_bucket_name: "{{ aws_bucket | default(clusterid ~ '-docker') }}"
```